apps/builder/src/features/blocks/inputs/fileUpload/components/FileInputSettings.tsx

```diff
@@ -134,6 +164,16 @@ export const FileInputSettings = ({ options, onOptionsChange }: Props) => {
                 withVariableButton={false}
               />
             </Stack>
+            {options?.allowedFileTypes && (
+              <TextInput
+                label={t("blocks.inputs.file.settings.allowedFileTypesLabel")}
+                defaultValue={
+                  options?.labels?.allowedFileTypes ??
+                  defaultFileInputOptions.labels.allowedFileTypes
+                }
+                onChange={updateAllowedFileTypesLabel}
+              />
+            )}
```

Comment:
Let's not bother with this, the file picker will make it explicit what is allowed